### PR TITLE
Add template for documentation issues & suggestions

### PR DESCRIPTION
**Pull request by townsend2010**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation-issue-suggestion.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: Documentation issue/suggestion
+about: Report issues or make suggestions about documentation
+title: ''
+labels: documentation
+assignees: ''
+
+---
+
+**What are issue or suggestion are you reporting?**
```
Comment (ricab) [inline .github/ISSUE_TEMPLATE/documentation-issue-suggestion.md:10]:
```suggestion
**What issue or suggestion are you reporting?**
```

Although I would just say "Description" in this title.

**Review by ricab — COMMENTED** [review]
Typo and suggestion, otherwise looks good.

File: .github/ISSUE_TEMPLATE/documentation-issue-suggestion.md (hunk 1 above)
Comment (townsend2010) [reply .github/ISSUE_TEMPLATE/documentation-issue-suggestion.md]: Thanks! I think I softened the language a bit.  Let me know what you think.

File: .github/ISSUE_TEMPLATE/documentation-issue-suggestion.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+name: Documentation issue/suggestion
+about: Report issues or make suggestions about documentation
+title: ''
+labels: documentation
+assignees: ''
+
+---
+
+**Please describe the issue or suggestion are you reporting**
```
Comment (ricab) [inline .github/ISSUE_TEMPLATE/documentation-issue-suggestion.md:10]:
```suggestion
**Please describe the issue or suggestion you are reporting**
```

**Review by ricab — COMMENTED** [review]
It looks like there's a new typo now.

To expand a little on my earlier suggestions of shorter titles (e.g. "Description"), there are two reasons why I thought they would be preferable: 

- once the report is submitted, anyone reading will see "please describe...", which doesn't really apply to the reader and requires some more scanning to figure out what is what;
- the person filling in the report already sees a very similar long version of what is expected on the following line.

**Comment by townsend2010** [comment]
Hi @ricab!

I guess what I'm trying to do with the headings is follow how we ask a user to report a feature: https://github.com/canonical/multipass/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=

But I get what you are saying and I'll see if I can make it better.

**Comment by townsend2010** [comment]
Ok, I *think* I made this better.  Let me know what you think!  Thanks!

**Review by ricab — APPROVED** [review]
Alright, I'm fine with that.

Should we bypass bors?

**Comment by ricab** [comment]
bors cancel

**Comment by townsend2010** [comment]
Yeah, we can bypass bors here.

**Comment by townsend2010** [comment]
Thanks for the review @ricab!

**Comment by ricab** [comment]
OK, just trying to get a message.

bors r+

**Comment by ricab** [comment]
bors cancel

**Comment by ricab** [comment]
Sure thing @townsend2010, thank you for the PR!

**Pull request merged**